src/bundles/notify.js

```diff
@@ -59,6 +59,15 @@ const notify = {
       }
     }
 
+    if (action.type === 'EXPERIMENTS_TOGGLE_FAIL') {
```

Comment:
should we add a success notification too, or do you think that would be too noisy?

Reply:
I think it'd be too noisy. The checkbox does the trick already

Reply:
agree, we will likely change these to switches in the future which will improve the UI feedback anyway.